docs/deployment/docker.md

```diff
@@ -197,12 +197,13 @@ FROM node:20-bookworm-slim AS build
 
 # Set Python interpreter for `node-gyp` to use
 ENV PYTHON=/usr/bin/python3
+ENV NODE_OPTIONS="--max-old-space-size=8192"
 
 # Install isolate-vm dependencies, these are needed by the @backstage/plugin-scaffolder-backend.
 RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     --mount=type=cache,target=/var/lib/apt,sharing=locked \
     apt-get update && \
-    apt-get install -y --no-install-recommends python3 g++ build-essential && \
+    apt-get install -y --no-install-recommends python3 g++ build-essential git && \
```

Comment:
We intentionally don't include `git` as it's not a requirement for Backstage to function, if you need it for your specific needs feel free to add it to your Dockerfile 👍